server/src/main/java/org/candlepin/async/tasks/HypervisorUpdateJob.java

```diff
@@ -120,16 +119,6 @@ public Object execute(final JobExecutionContext context) throws JobExecutionExce
                 return "Nothing to do. Owner does not exist";
             }
 
-            if (owner.isAutobindDisabled() || owner.isContentAccessEnabled()) {
```

Comment:
Are we sure we want to completely remove the 'owner autobind disabled' functionality? I see that the change (43c11bac9095d0913414c77af5b7235947b87e36) that introduced the golden ticket restriction only added the second part of the if statement, while the `owner.isAutobindDisabled()` was still there.

Reply:
For the sake of creating the hypervisor/guest relationship, it makes no sense to block for autobind disabled.

This is left behind from when we handled guest migrations differently.

Reply:
ack